package.json

```
@@ -16,15 +16,16 @@
   "main": "lib/index.js",
   "bin": "lib/bin/analyzer.js",
   "engines": {
-    "node": ">= 6.14.4"
+    "node": ">= 10.13.0"
   },
   "scripts": {
     "start": "gulp watch",
     "build": "gulp build",
     "npm-publish": "npm run lint && npm run build && npm test && npm publish",
     "lint": "eslint --ext js,jsx .",
-    "test": "mocha --exit --require @babel/register",
-    "test-dev": "mocha --watch --require @babel/register"
+    "install-test-webpack-versions": "./bin/install-test-webpack-versions.sh",
+    "test": "npm run install-test-webpack-versions && mocha --exit --require @babel/register",
+    "test-dev": "npm run install-test-webpack-versions && mocha --watch --require @babel/register"
```

The `mocha --watch` now is constantly re-running the tests when `npm run test-dev` is started.

It seems that this is caused by the support for testing different webpack versions and those tests outputting files to `test/output` directory, triggering mocha's watcher to rerun the tests as soon as they are finished.

I was able to leverage `--watch-ignore` flag as [described in the docs](https://mochajs.org/#-watch-w) to make this `test-dev` stop until changes are done to some tests ☺️ 

```suggestion
    "test-dev": "npm run install-test-webpack-versions && mocha --watch --watch-ignore test/output --require @babel/register"
```

---

Nice catch! Fixed